app/src/main/java/ca/unb/mobiledev/reflexrevolution/GameActivity.java

```diff
@@ -0,0 +1,162 @@
+package ca.unb.mobiledev.reflexrevolution;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.CountDownTimer;
+import android.view.View;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class GameActivity extends AppCompatActivity {
+    //Time in ms
+    //Could get rid of this and do random intervals or scale it off of score
+    private final int TIME_BETWEEN_LOOPS = 1000;
+
+    private TextView timeText;
+    private TextView scoreText;
+    private LinearLayout layout; //Layout we should add new UI elements to
+
+    private CountDownTimer timer;
+    private String currentInstruction = null;
+    private ArrayList<String> instructions;
```

Comment:
I think defining a custom enum would make the code clearer than using strings. We can simply define the public enum in its own file. Then we can use this new `Instruction` type instead of string, and use a nice `switch()` statement instead of a bunch of if-else inside the `displayInstruction()` method.
```java
package ca.unb.mobiledev.reflexrevolution;

public enum Instruction {
    NONE,
    TAP,
    SWIPE,
    SHAKE
    // Will add more later
}
```
```java
switch(currentInstruction) {
    case TAP:
        // Do stuff
        break;
    case SWIPE:
        // Do stuff
        break;
    case SHAKE:
        // Do stuff
        break;
}
```

Reply:
Yeah I like the idea of an enum better too, good plan!